bucket/msiafterburner.json

```diff
@@ -1,52 +1,50 @@
     "homepage": "https://www.msi.com/page/afterburner",
     "description": "Overclocking utility for graphics cards. Main features include GPU/Shader/Memory clock adjustment, advanced fan speed and GPU voltage control.",
     "license": {
         "identifier": "Freeware",
         "url": "https://www.msi.com/page/website-terms-of-use"
     },
-    "version": "4.6.0",
-    "url": "http://download.msi.com/uti_exe/vga/MSIAfterburnerSetup.zip#/MSIAfterburnerSetup460.7z",
-    "hash": "dd8be977fdde25343f8db3e318af4a5449aa90453505b78fc8f5975cddd98311",
+    "version": "4.6.1",
+    "url": "https://download-eu2.guru3d.com/afterburner/%5BGuru3D.com%5D-MSIAfterburnerSetup461Build15561.zip#/sourceforge.net.zip",
+    "hash": "4e59da5b73c5f290408851fea6e48b3b37dd6b089ae5598b3ccee43ea11fe603",
     "suggest": {
-        "Visual C++ Redist 2008": "extras/vcredist2008"
+        "Visual C++ Redist 2008": "extras/vcredist2008",
+        "RivaTuner Statistics Server": "extras/rtss",
+        "MSI Kombustor": "extras/msikombustor",
+        "FurMark": "extras/furmark"
     },
-    "persist": [
-        [
-            "Profiles",
-            "AB_Profiles"
-        ],
-        [
-            "RTSS/Profiles",
-            "RTSS_Profiles"
-        ]
-    ],
-    "bin": [
-        "RTSS/RTSS.exe",
-        "MSIAfterburner.exe"
-    ],
+    "persist": "Profiles",
+    "bin": "MSIAfterburner.exe",
     "pre_install": [
-        "Expand-7zipArchive \"$dir\\MSIAfterburnerSetup*.exe\" \"$dir\"",
-        "Expand-7zipArchive \"$dir\\Redist\\RTSSSetup.exe\" \"$dir\\RTSS\"",
-        "Get-ChildItem $dir -Recurse -Include @('MSIAfterburnerSetup*.exe', '$PLUGINSDIR', 'Uninstall.exe.nsis', '$R0') | Remove-Item -Recurse",
-        "Move-Item $dir/RTSS/RTSSHooks.dll.copy $dir/RTSS/RTSSHooks.dll",
-        "Move-Item $dir/RTSS/RTSSHooks64.dll.copy $dir/RTSS/RTSSHooks64.dll"
+        "Expand-7zipArchive \"$dir\\MSIAfterburnerSetup*.exe\" \"$dir\" -Removal",
+        "Get-ChildItem \"$dir\" -Recurse -Include @('$PLUGINSDIR', 'Uninstall.exe.nsis', '$R0', 'Redist\\*.exe') | Remove-Item -Recurse",
+    ],
+    "post_install": [
+        "if(Test-Path \"$persist_dir\\AB_Profiles\") {",
+        "    warn 'Migrating Afterburner profiles...'",
+        "    Move-Item -Path \"$persist_dir\\AB_Profiles\\*\" -Destination \"$persist_dir\\Profiles\\\"",
+        "    Remove-Item \"$persist_dir\\AB_Profiles\"",
+        "}"
     ],
     "shortcuts": [
         [
             "MSIAfterburner.exe",
             "MSI Afterburner"
-        ],
-        [
-            "RTSS/RTSS.exe",
-            "RivaTuner Statistics Server"
         ]
     ],
     "checkver": {
-        "url": "http://event.msi.com/afterburner/Update.txt",
-        "re": "ProductBetaVersion\\s+=\\s+0.*\\nProductVersion\\s+=\\s+([\\d\\.]+)"
+        "url": "https://www.guru3d.com/files-details/msi-afterburner-beta-download.html",
```

Comment:
Beta?

Reply:
Yep, the page is called beta because it also lists 4.6.2 Beta. The download section below shows 4.6.1 as Stable/Final.